pkg/skaffold/output/output.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package output
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+type SkaffoldWriter struct {
+	MainWriter  io.Writer
+	EventWriter io.Writer
+}
+
+func (s SkaffoldWriter) Write(p []byte) (int, error) {
+	for _, w := range []io.Writer{s.MainWriter, s.EventWriter} {
+		n, err := w.Write(p)
+		if err != nil {
+			return n, err
```

Comment:
Should we ignore errors on the non-MainWriter?

Reply:
I could see the case for that. I'll change this to focus the checks on the mainwriter